Dockerfile

```diff
@@ -1,45 +1,13 @@
 FROM node:6.11.1-alpine
 
-RUN apk update && apk upgrade
-
-# Install packages needed for production
-RUN apk add --update bash python make g++
-
-# Install packages needed for testing
-RUN apk add --update ruby openssl
-
-# Install glibc (https://github.com/sgerrand/alpine-pkg-glibc), needed to run pact-mock_service
-# (https://github.com/bethesque/pact-mock_service) which ships with a binary dependency (@pact-foundation/pact-mock-service-linux-x64)
-# linked against glibc.
-ADD docker/sgerrand.rsa.pub /etc/apk/keys/sgerrand.rsa.pub
-RUN apk --no-cache add ca-certificates
-RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.25-r0/glibc-2.25-r0.apk
-RUN apk add glibc-2.25-r0.apk
+RUN apk update &&\
+    apk upgrade &&\
+    apk add --update libc6-compat
 
 ENV PORT 9000
-
 EXPOSE 9000
 
-# add package.json before source for node_module cache
-ADD package.json /tmp/package.json
-RUN cd /tmp && npm install
-
-ADD . /app
 WORKDIR /app
+ADD . /app
```

Comment:
this is all great stuff. Looking at it again it seems like we are still copying too much into the container - like here we are copying all the tests into the prod container, and I think other things (eg. CONTRIBUTING.md)